src/com/codeenginestudio/bookManagement/servlets/EditBookServlet.java

```diff
@@ -11,54 +11,37 @@
 
 import com.codeenginestudio.bookManagement.manage.ManageBook;
 import com.codeenginestudio.bookManagement.models.Book;
+import com.codeenginestudio.bookManagement.util.BookUtil;
+import com.codeenginestudio.bookManagement.validator.BookValidator;
 
-/**
- * Servlet implementation class EditBookServlet
- */
 @WebServlet("/EditBookServlet")
 public class EditBookServlet extends HttpServlet {
 
-    public void displayView(HttpServletRequest request, HttpServletResponse response, String fileName)
-			throws ServletException, IOException {
-
-		RequestDispatcher dispatcher = this.getServletContext().getRequestDispatcher(fileName);
-		dispatcher.forward(request, response);
-	}
-
-	/**
-	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
-	 */
 	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
 
 		int bookId = Integer.parseInt(request.getParameter("bookId"));
-		request.setAttribute("url", "/Book/EditBook");
-		request.setAttribute("book", manageBook.getOneBook(bookId));
-		displayView(request, response, "/view/AddOrEditBook.jsp");
+		request.setAttribute("book", ManageBook.getOneBook(bookId));
+		BookUtil._displayView(request, response, "/view/AddOrEditBook.jsp");
 	}
 
-	/**
-	 * @see HttpServlet#doPost(HttpServletRequest request, HttpServletResponse response)
-	 */
 	protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
 
 		int bookId = Integer.parseInt(request.getParameter("bookId"));
 		String bookName = request.getParameter("bookName");
 		String bookAuthor = request.getParameter("bookAuthor");
 
-		if(bookName.equals("")) {
+		if(BookValidator._checkNullValue(bookName)) {
```

Comment:
The method name should be `BookValidator.validate(bookName)`, and in that method, you have a `_checkNull` method. It should return a list of errors. If the list is empty, mean no errors. It helps you add more validation methods later easier.